aspire4j/src/main/java/com/microsoft/aspire/components/common/traits/ResourceWithBindings.java

```diff
@@ -8,8 +8,8 @@ public interface ResourceWithBindings<T extends ResourceWithBindings<T>> {
     T withBinding(Binding binding);
 
     default T withExternalHttpEndpoints() {
-        withBinding(new Binding(Binding.Scheme.HTTP, Binding.Protocol.TCP, Binding.Transport.HTTP).withExternal());
-        withBinding(new Binding(Binding.Scheme.HTTPS, Binding.Protocol.TCP, Binding.Transport.HTTP).withExternal());
+        withBinding(new Binding(Binding.Scheme.HTTP, Binding.Protocol.TCP, Binding.Transport.HTTP).withTargetPort(8080).withExternal());
+        withBinding(new Binding(Binding.Scheme.HTTPS, Binding.Protocol.TCP, Binding.Transport.HTTP).withTargetPort(8080).withExternal());
```

Comment:
This very gross hack was introduced so the manifest would have a `targetPort` element, which is needed for `dockerfile.v0` and `container.v0` resources in the manifest.  This isn't the right place to add it in general, but there is exactly one caller of this method right now so I took a shortcut.